notebooks/src/el-paso-workflow.ipynb

```diff
@@ -30,58 +31,71 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "import ergo\n",
-    "import seaborn\n",
-    "\n",
-    "import numpy as np\n",
     "import warnings\n",
     "import ssl\n",
-    "import pandas as pd\n",
-    "from datetime import timedelta, date\n",
-    "\n",
     "warnings.filterwarnings(action=\"ignore\", category=FutureWarning)\n",
     "warnings.filterwarnings(action=\"ignore\", module=\"plotnine\")\n",
-    "ssl._create_default_https_context = ssl._create_unverified_context\n",
-    "pd.set_option('precision', 2)"
+    "ssl._create_default_https_context = ssl._create_unverified_context"
    ]
   },
   {
-   "cell_type": "markdown",
+   "cell_type": "code",
+   "execution_count": null,
    "metadata": {},
+   "outputs": [],
    "source": [
-    "# Forecasting with Ergo"
+    "import ergo\n",
+    "import seaborn\n",
+    "\n",
+    "import numpy as np\n",
+    "import pandas as pd\n",
+    "from datetime import timedelta, date\n",
+    "import matplotlib.pyplot as plt\n",
+    "\n",
+    "pd.set_option('precision', 2)\n",
+    "\n",
+    "def summarize_samples(samples):\n",
+    "    stats = samples.describe(percentiles=[0.05, 0.5, 0.95])\n",
+    "    percentile = lambda pt: float(stats.loc[f\"{pt}%\"])\n",
+    "    return f\"{percentile(50):.2f} ({percentile(5):.2f} to {percentile(95):.2f})\"\n",
+    "\n",
+    "def show_marginal(func):\n",
+    "    samples = ergo.run(func, num_samples=1000)\n",
+    "    seaborn.distplot(samples).set_title(func.__doc__);\n",
+    "    plt.show()\n",
+    "    print(f\"Median {func.__doc__}: {summarize_samples(samples)}\")"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## Choose a decision-relevant question"
+    "# Choose a decision-relevant question"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### How many ventilators will be needed in El Paso?\n",
+    "## How many ventilators will be needed in El Paso?\n",
     "\n",
-    "I want to predict this question:\n",
+    "I want to predict on this question:\n",
     "\n",
-    "> How many patients with Covid-19 in El Paso County will require invasive ventilation on the day when the number of hospital admissions of cases peak?\n",
+    "> How many patients with Covid-19 in El Paso County will require invasive ventilation on the day when the number of hospital patients of cases peak?\n",
```

Comment:
I prefer to say "patients" rather than "admissions" because what we're talking about is "number of patients in the hospital/ICU/ventilators on a given day," not e.g. "number of patients who were admitted to the hospital on that day"

Reply:
"beds" could also be good ("hospital beds", "ICU beds")